### ISSUE TITLE
[Multi_scan] quit_simulators not working in latest releases

### ISSUE THREAD

**Issue by szymon-kazmierczak** [lyndsey-ferguson/fastlane-plugin-test_center#140]
### New Issue Checklist

- [x] Updated `fastlane-plugin-test_center` to the latest version
- [x] I read the [README.md](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md)
- [x] I reviewed the [example(s)](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md) for the action(s) I am using
- [x] I have removed any sensitive data such as passwords, authentication tokens, or anything else I do not want to world to see

### Issue Description
test_center v3.8.3, multi_scan.

Quit_simulators argument is not showing up in arguments list and is not working as expected (simulator killed with quit_simulators: false).
 
+-------------------------+-------------------------------------------------+
|                Summary for multi_scan (test_center v3.8.3)                |
+-------------------------+-------------------------------------------------+
| try_count               | 3                                               |
| testrun_completed_block | #<Proc:0x00007ff6d1c09fc0@Fastfile:53 (lambda)> |
+-------------------------+-------------------------------------------------+

According to another user, everything worked fine in 3.7.0. https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/pull/91#issuecomment-524305292
### Environment

<details>
  <pre>### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_IE.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
opt_out_usage

# If you want to automatically update fastlane if a new version is available:
#update_fastlane

# This is the minimum version number required.
# Update this, if you use features of a newer version
fastlane_version "2.107.0"

XCODE_WORKSPACE_FILEPATH = File.absolute_path('../PROJECT.xcworkspace')

default_platform :ios

platform :ios do

  before_all do
    cocoapods(
      try_repo_update_on_error: true,
      use_bundle_exec: true
    )

    xcversion(version: "~> 11.0")
  end

  lane :testUI do |options|
    ENV['FASTLANE_EXPLICIT_OPEN_SIMULATOR'] = "true"
    launch_simulator(device, "iOS-12-2", device)

    test_run_block = lambda do |testrun_info|
      failed_test_count = testrun_info[:failed].size
      if failed_test_count > 0
        UI.important("There were #{failed_test_count} failed tests.")
        try_attempt = testrun_info[:try_count]
        if try_attempt < 3
          UI.header("Rerun Attempt: #{try_attempt}")
        end
      end
    end

    result = multi_scan(
      workspace: XCODE_WORKSPACE_FILEPATH,
      try_count: 3,
      fail_build: false,
      scheme: scheme,
      testrun_completed_block: test_run_block,
      device: device,
      output_directory: "fastlane/reports",
      result_bundle: "TestResults",
      prelaunch_simulator: true,
      quit_simulators: false
    )
    unless result[:failed_testcount].zero?
      UI.message("There are #{result[:failed_testcount]} legitimate failing tests")
    end
  end

```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.131.0 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

| Plugin                      | Version | Update-Status |
| --------------------------- | ------- | ------------- |
| fastlane-plugin-trainer     | 0.4.1   | ✅ Up-To-Date  |
| fastlane-plugin-test_center | 3.8.3   | ✅ Up-To-Date  |</pre>
</details>

**Comment by szymon-kazmierczak** [comment]
I didn't paste in my fastlane output as it contains a lot of sensitive data and does not seem relevant to the bug report. Let me know what you think and if i can help in any way! Love the project, and would love to help out

**Comment by lyndsey-ferguson** [comment]
Are you certain it is not quitting simulators? I have [code](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/lib/fastlane/plugin/test_center/actions/multi_scan.rb#L14) that does it directly rather than using `fastlane's` version.

**Comment by szymon-kazmierczak** [comment]
It's killing them when it shouldn't is what i wanted to say! I set my `quit_simulators: false` expecting the simulators to stay alive

**Comment by lyndsey-ferguson** [comment]
# 😨

**Comment by szymon-kazmierczak** [comment]
I have a feeling changes of the multi_scan.rb in this commit may be the culprit?

https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/commit/bbf1637943b289261bfc693d76df0db8edf254dc#diff-b1627ddcb767b566591f53bc2ae1fcffR11

**Comment by lyndsey-ferguson** [comment]
That's it, yes.

**Comment by lyndsey-ferguson** [comment]
Can you modify your `Pluginfile` per my instructions below, run `bundle install`, and then run your fastlane again?

Pluginfile:
```
gem 'fastlane-plugin-test_center', :git => "https://github.com/lyndsey-ferguson/fastlane-plugin-test_center.git", :branch => "issue-140-quitting-sims-unwanted"
```

Please attach the console output as a text file to this issue (makes it easier for me to review).

**Comment by lyndsey-ferguson** [comment]
Fixed in [v3.8.4](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/releases/tag/v3.8.4)
